test/wpool_process_SUITE.erl

```diff
@@ -101,3 +105,34 @@ call(_Config) ->
   ok3 = wpool_process:call(Pid, {stop, normal, ok3, state}, 5000),
   timer:sleep(1000),
   false = erlang:is_process_alive(Pid).
+
+-spec pool_restart_crash(config()) -> _.
+pool_restart_crash(_Config) ->
+    Pool = pool_restart_crash,
+    {ok, Pid} = wpool:start_pool(Pool, [{workers, 2}, {worker, {crashy_server, []}}]),
```

Comment:
I'm not sure why @gadgetci is not warning us about this line, but when I run `$ rebar3 ct` or `$ elvis rock` on my computer, I do get warnings about it.
Can you reduce it to 80 characters, please?